feat(docker): add more pruning aliases to docker plugin

augustfengd opened the pull request:
## Standards checklist:

<!-- Fill with an x the ones that apply. Example: [x] -->

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- add some more pruning commands to docker plugin

## Other comments:

There is already an existing alias for pruning, and this commit introduces some more pruning commands that are consistent with the existing one.

JimMoen commented:
please update docs page [docker/README.md](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/docker/README.md)

augustfengd commented:
@JimMoen I added another alias (`dprune`) which combines all the prune commands together and updated the docs. Please let me know what you think, thanks!

alvincrespo commented on plugins/docker/docker.plugin.zsh:
```diff
@@ -28,7 +28,11 @@ alias dstp='docker container stop'
+alias diprune="docker image prune"
+alias dnprune="docker network prune"
 alias dvprune='docker volume prune'
+alias dprune="dcprune; diprune; dnprune; dvprune"
```
@augustfengd Wouldn't `docker system prune -a` be more appropriate here?

alvincrespo replied:
ref: https://docs.docker.com/reference/cli/docker/system/prune/

augustfengd replied:
Yes! This makes sense. I've updated my PR to do that.

augustfengd replied:
I also grouped the aliases by command so the diff is a bit noisy now.

alvincrespo reviewed (approved):
This looks awesome! We should definitely get this in 😄

alvincrespo commented:
@carlosala Would it be possible to get this one merged in?

baltic-tea commented:
Hi, @augustfengd! Create a pull request with your changes on my branch if you want: https://github.com/ohmyzsh/ohmyzsh/pull/11821

Maybe someday all of this will be merged 🙏
